firstvoices/backend/tasks/mtd_export_tasks.py

```diff
@@ -188,23 +190,27 @@ def check_sites_for_mtd_sync(self):
     try:
         self.state = "STARTED"
         sites = Site.objects.filter(visibility=constants.Visibility.PUBLIC)
-        six_hours_ago = timezone.now() - timedelta(hours=6)
+        delta = get_delta_for_mtd_check() or timedelta(hours=24)
+        last_modified_check_period = timezone.now() - delta
```

Comment:
I can think of a few cases where this will have trouble, for example:

* We manually generated an MTD export a long time ago, then edited some data, and just now started running this task. The old changes will not be noticed because they didn't happen during the calculated time delta.

Could we adjust this logic to be more like: check the timestamp on the most recent MTD export, and if it is longer ago than some configurable minimum time (e.g., 1 hour), check for data that has changed since then? It is more meaningful to check the timestamp on the exported data rather than the last time the task ran, and probably easier as well.